charts/cloudzero-agent/templates/insights-deploy.yaml

```diff
@@ -37,6 +37,15 @@ spec:
         runAsNonRoot: true
         runAsGroup: 65534
         fsGroup: 65534
+      affinity:
```

Comment:
This is to make sure we don't schedule all pods on the same node right?

Reply:
Yes - feedback from DVAG was we should add this... to ensure the replicaset it started on different nodes for each pod (if possible)